osu.Game.Rulesets.Mania/UI/Column.cs

```diff
@@ -44,6 +44,63 @@ public class Column : ScrollingPlayfield, IKeyBindingHandler<ManiaAction>, IHasA
 
         private readonly GameplaySampleTriggerSource sampleTriggerSource;
 
+        public class ColumnTouchInputArea : Container
+        {
+            private Column column => (Column)Parent;
+
+            private Container hintContainer;
+
+            public ColumnTouchInputArea()
+            {
+                RelativeSizeAxes = Axes.X;
+                Anchor = Anchor.BottomCentre;
+                Origin = Anchor.BottomCentre;
+                Height = 100;
+                InternalChild = hintContainer = new Container
+                {
+                    RelativeSizeAxes = Axes.Both,
+                    BorderColour = Color4.Red,
+                    BorderThickness = 5,
+                    Masking = true,
+                };
+            }
+
+            protected override void LoadComplete()
+            {
+                hintContainer.Delay(1000).FadeOutFromOne(500, Easing.OutSine);
+            }
+
+            private ManiaInputManager.RulesetKeyBindingContainer keyBindingContainer { get; set; }
+
+            private ManiaInputManager.RulesetKeyBindingContainer getKeyBindingContainer()
+            {
+                return keyBindingContainer ??= (ManiaInputManager.RulesetKeyBindingContainer)((ManiaInputManager)GetContainingInputManager()).KeyBindingContainer;
+            }
+
+            protected override bool OnTouchDown(TouchDownEvent e)
+            {
+                getKeyBindingContainer().TriggerPressed(column.Action.Value);
+                return base.OnTouchDown(e);
```

Comment:
Have you tested this? I am very sure this will not work how you expect it to with multiple touches at once, due to the fact you aren't returning `true`. This would cause the event to also propagate as a (fallback) mouse click event, and probably break everything.